mw-webapp/src/component/icon/Icon.tsx

```diff
@@ -137,6 +138,11 @@ export const IconDictionary = {
    * User icon
    */
   UserIcon: (params: IconProps) => <UserIcon {...params} />,
+
+  /**
+   * User icon
```

Comment:
!

Reply:
done

Reply:
Maybe i see old commit but I also see User instead of Chevron on 149 line)